lib/tasks/user.rake

```diff
@@ -0,0 +1,38 @@
+namespace :hyku do
+  namespace :superadmin do
+    desc 'Create an initial superadmin user'
+    task create: :environment do
+      puts 'Creating an initial superadmin user.'
+      user = prompt_to_create_user
+      user.add_role(:superadmin)
+      puts 'User created.'
+    end
+
+    def prompt_to_create_user
```

Comment:
@jcoyne If it only creates new users, do we need `#find_or_create_by` on line 12? The `find` there suggests that this task can act on existing users.